gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1020,6 +1020,7 @@ double ofx_get_investment_amount(const struct OfxTransactionData* data)
     }
 }
 
+// Let the user selecct a ofx qfx file and start the import
```

Comment:
This comment is superfluous, the function is documented in gnc-ofx-import.h. Besides, you mis-spelled 'select'.